src/include/ceph_fs.h

```diff
@@ -457,13 +457,16 @@ extern const char *ceph_mds_op_name(int op);
 #define CEPH_O_RDONLY          00000000
 #define CEPH_O_WRONLY          00000001
 #define CEPH_O_RDWR            00000002
+#define CEPH_O_FMODE_EXEC      00000040
 #define CEPH_O_CREAT           00000100
 #define CEPH_O_EXCL            00000200
 #define CEPH_O_TRUNC           00001000
 #define CEPH_O_LAZY            00020000
 #define CEPH_O_DIRECTORY       00200000
 #define CEPH_O_NOFOLLOW        00400000
 
+#define O_FMODE_EXEC	       00000040
```

Comment:
```suggestion
#define O_FMODE_EXEC	       CEPH_O_FMODE_EXEC
```

Value of `FMODE_EXEC` whether for kernel or libcephfs would be same to maintain uniformity right? Should doing this make sense?

Reply:
yeah, I just copied this from Linux Kernel VFS. And the VFS will pass this flag to lower level layer, which is the FUSE and then FUSE will pass it to the userspace.